hawkauthlib/utils.py

```diff
@@ -35,7 +34,7 @@ def b64encode(data):
 
 else:  # pragma: nocover
 
-    bytes = str
+    bytes = str    # pylint: disable=W0622
```

Comment:
There's an inconsistent number of spaces before `# pylint` on these two lines, and I'm surprised that the linter didn't complain about it :-)

Reply:
how should such an assessment fail? .. I mean; these are just two comments 12 lines apart from each other.

Reply:
Well, `flake8` regularly complains to me about stylistic issues in my placement of comments, e.g. not using at least two spaces before `#`, or one space after it.  I dont think that's necessarily a valuable thing for it to complain about, I was just surprised that pylint wasn't doing the same thing.